thefuck/rules/git_clone_missing.py

```diff
@@ -0,0 +1,33 @@
+'''
+Rule: git_clone_missing
+
+Correct missing `git clone` command when pasting a git URL
+
+```sh
+>>> https://github.com/nvbn/thefuck.git
+git clone https://github.com/nvbn/thefuck.git
+```
+
+Author: Miguel Guthridge
+'''
+from six.moves.urllib import parse
+
+
+def match(command):
+    script = command.script
+    parts = command.script_parts
+    # We want it to be a URL by itself
+    if len(parts) > 1:
```

Comment:
No need to create a new variable, that's readable enough:
```suggestion
    if len(command.script_parts) > 1:
```